devices/Cargo.toml

```diff
@@ -13,12 +13,15 @@ byteorder = "1.4.3"
 hypervisor = { path = "../hypervisor" }
 libc = "0.2.135"
 log = "0.4.17"
+phf = { version = "0.11", features = ["macros"] }
+thiserror = "1.0.30"
 versionize = "0.1.6"
 versionize_derive = "0.1.4"
 vm-device = { path = "../vm-device" }
 vm-memory = "0.9.0"
 vm-migration = { path = "../vm-migration" }
 vmm-sys-util = "0.10.0"
+tpm = { path = "../tpm" }
```

Comment:
Something has gone wrong with the alphabetical order here.

Reply:
this went haywire when I renamed vtpm -> tpm. fixed now.

Reply:
This doesn't look like it got fixed?

Reply:
Sorry about this. Looks like I dropped this change accidentally before I pushed. Correctly fixed now.